homeassistant/components/camera/strings.json

```diff
@@ -46,6 +46,10 @@
           }
         }
       }
+    },
+    "legacy_webrtc_provider": {
+      "title": "Detected use of legacy WebRTC provider registered by {legacy_integration}",
+      "description": "The {legacy_integration} integration has registered a legacy WebRTC provider. Home Assistant prefers using the built-in modern WebRTC provider registered by the {builtin_integration} integration.\n\nBenefits of the built-in integration are:\n\n- The camera stream is started faster.\n- More camera devices are supported.\n\nTo fix this issue, you can either keep using the built-in modern WebRTC provider and remove the {legacy_integration} integration or remove the {builtin_integration} integration to use the legacy provider, and then restart Home Assistant."
```

Comment:
I don't say here exactly how to remove the built-in go2rtc integration. I think that can be explained in the go2rtc docs, which are linked in the read-more url.